src/Symfony/Component/Config/Util/XmlUtils.php

```diff
@@ -51,27 +51,17 @@ public static function parse(string $content, $schemaOrCallable = null)
         }
 
         $internalErrors = libxml_use_internal_errors(true);
-        if (\LIBXML_VERSION < 20900) {
-            $disableEntities = libxml_disable_entity_loader(true);
-        }
         libxml_clear_errors();
 
         $dom = new \DOMDocument();
         $dom->validateOnParse = true;
-        if (!$dom->loadXML($content, \LIBXML_NONET | (\defined('LIBXML_COMPACT') ? \LIBXML_COMPACT : 0))) {
-            if (\LIBXML_VERSION < 20900) {
-                libxml_disable_entity_loader($disableEntities);
-            }
-
+        if (!$dom->loadXML($content, \LIBXML_NONET | \LIBXML_COMPACT)) {
```

Comment:
💡 `LIBXML_COMPACT` is defined for libXML 2.6.21 and newer, according to https://www.php.net/manual/en/libxml.constants.php